adafruit_connection_manager.py

```diff
@@ -383,10 +373,7 @@ def connection_manager_close_all(
 
 def get_connection_manager(socket_pool: SocketpoolModuleType) -> ConnectionManager:
     """
-    Get the ConnectionManager singleton for the given pool.
-
-    - **socket_pool** *(Optional[SocketpoolModuleType])* – The SocketPool you want the
-      ConnectionManager for
+    Get or create the ConnectionManager singleton for the given pool.
```

Comment:
Curious, why the removal of the `param`?

Reply:
because the parameter is already explained in the first sentence. I started writing a `:param ...` and the text there was just redundant.